src/app/dashboard/roles/page.tsx

```
@@ -22,17 +79,64 @@ export default function RolesPage() {
+    );
+  }
+
+  if (error) {
+    return (
+      <main className="pl-[260px] pr-6 pt-6">
+        <div className="flex flex-col items-center justify-center h-64">
+          <div className="text-red-600 mb-4">{error}</div>
+          <button
+            onClick={fetchRoles}
+            className="bg-amber-600 text-white px-4 py-2 rounded-lg hover:bg-amber-700"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)